esp/services/espcontrol/ws_espcontrolservice.cpp

```diff
@@ -112,18 +112,28 @@ bool CWSESPControlEx::handleDaliAttachmentRequest(bool attach, bool force, Strin
 
 bool CWSESPControlEx::onDetachFromDali(IEspContext& context, IEspDetachFromDaliRequest& req, IEspDetachFromDaliResponse& resp)
 {
+    bool status = false;
+#ifdef _CONTAINERIZED
+    const char *message = "Dynamically detaching ESP from DALI is a baremetal only feature";
```

Comment:
I am curious why it is bare-metal only.  Also, could code this to remove the #ifdef

    StringBuffer message;
    if (isContainerized())
        message.append("Dynamically detaching ESP from DALI is a baremetal only feature");
    else
        status = handleDaliAttachmentRequest(false, req.getForce_isNull() ? false : req.getForce(), message);

Reply:
It's really by choice.  I want to deprecate the feature if I can.  With this change (especially in the cloud) there are better ways of managing these services without DALI and therefore no need to attach/detatch.

I'll change it not to use the #ifdef though.